pkg/controller/cronjob/utils.go

```diff
@@ -183,6 +183,7 @@ func getFinishedStatus(j *batchv1.Job) (bool, batchv1.JobConditionType) {
 	return false, ""
 }
 
+// IsJobFinished returns whether or not a job has finished.
```

Comment:
Might be worth mentioning that its based on if its been marked as finished irrespective of it it was successfully.